packages/cdktf-cli/bin/cmds/helper/init.ts

```diff
@@ -113,7 +123,7 @@ This means that your Terraform state file will be stored locally on disk in a fi
 
     // Check if token is set so we can set up Terraform Cloud workspace
     // only set with the '--local' option is specified the user.
```

Comment:
That comment seems vastly outdated now (and incorrect even before that 😅)
```suggestion
    // Set up a Terraform Cloud workspace if the user opted-in
```